static/css/styles.css

```diff
@@ -0,0 +1,23 @@
+body {
+    margin: 0;
+}
+
+.sidebar-nav {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  background-color: #333333;
+}
+
+.ref {
+  display: block;
+  color: white;
+  text-align: center;
+  padding: 16px;
+  float: left;
+}
+
+a {
```

Comment:
It's doesn't seem like a good practice to make all the links white. I think it's better to introduce a class for navigation links.

Reply:
What about this?